pallets/runtime/tests/src/identity_test.rs

```diff
@@ -1266,6 +1266,18 @@ fn changing_primary_key_we() {
     let charlie = AccountKeyring::Charlie;
     assert_ok!(accept(charlie, add(charlie)));
     assert_eq!(alice_pk(), charlie.to_account_id());
+
+    // Do it again but now making a secondary key the new primary key.
+    let join_auth = Identity::add_auth(
+        alice.did,
+        Signatory::Account(alice.acc()),
+        AuthorizationData::JoinIdentity(Permissions::default()),
+        None,
+    );
+    assert_ok!(Identity::join_identity(alice.origin(), join_auth));
+
+    assert_ok!(accept(alice.ring, add(alice.ring)));
```

Comment:
That comment had confused me at first.

```suggestion
    // Add Alice's old primary key as a secondary key to the identity..
    let join_auth = Identity::add_auth(
        alice.did,
        Signatory::Account(alice.acc()),
        AuthorizationData::JoinIdentity(Permissions::default()),
        None,
    );
    assert_ok!(Identity::join_identity(alice.origin(), join_auth));

    // Do it again but now making the secondary key the new primary key.
    assert_ok!(accept(alice.ring, add(alice.ring)));
```